theatres/chernarus-2024/TT30_Jonty_ChernarusTemplate.chernarus/edit_me/briefing/situation.html

```diff
@@ -0,0 +1,34 @@
+In late 2024, Chernarus was on the precipice of a pivotal moment in its history, with national government elections set to determine the course of the nation. However, instead of marking a democratic transition, these elections became the catalyst for a bitter conflict that would divide the nation. The conservative government, long in power, faced a challenge from a radical opposition party, whose members accused the incumbent administration of electoral fraud and corruption. This accusation sparked widespread discontent and ignited a political firestorm.
+
+The tension escalated when the leader of the radical opposition, Mikhail Volkov, leveraging support from a faction within the military known as the Red Star Alliance, orchestrated a swift takeover of the northern part of Chernarus. This move included securing key strategic assets such as airports and oil fields, effectively giving them control over the nation's trade routes, import and export channels, and a significant portion of its oil supply. This strategic advantage allowed the radicals to consolidate their power and extend their influence across the region.
+
+In response to this coup, the Chernarus Defense Forces (CDF) initiated operations aimed at reclaiming control over the country. Their objective was twofold: to retake the strategically vital areas lost to the radicals and to ensure conditions conducive to holding a fair re-election, which the radicals vehemently opposed. The CDF's efforts were met with fierce resistance, as the radicals sought to maintain their grip on power through force. The current situation in Chernarus is one of escalating conflict and uncertainty. The northern part of the country remains under the control of the radical faction, which continues to use its strategic advantages to bolster its position.
+
+Synixe Contractos are hired by the conservative party to support the CDF's fight to regain control and restore order, aiming to pave the way for a free and fair election that truly represents the will of the Chernarus people.
+
+<font color='#e3d310'>EMPLOYER</font>
+Name: Chernarussian Conservative Party
```

Comment:
That's extremely odd, we should either be working for the currently elected government or the military. Saying that we are working for a particular political party is unusual

Reply:
The conservative party is technically still in power since the elections never happened.  The radicals are almost conducting a coup since they aren't allowing a fair election to actually happen (got that idea from January 6th capitol attack). It's almost like good vs bad parties since NATO provided equipment when requested by the conservative party and the radicals just stole/assumed command over existing equipment. Plus the radical party is commanding a part of military that's essentially gone rogue against the chernarus military. I'll clarify it by changing the language a bit.

Reply:
> We are working WITH the military but we are HIRED by the conservative party

This is extremely unusual and definitely grounds for international condemnation, not NATO support. This is a sign of dictatorship, authoritarian, or corrupt regimes. We should be working with and hired by the military, or the official government. Politicians from parties form government, not a single party